.github/workflows/jlpkgbutler-ci-master-workflow.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        julia-version: ['1.0', '1.1', '1.2', '1.3', '1.4', '1.5']
+        julia-version: ['1.0', '1.1', '1.2', '1.3', '1.4', '1.5', 'nightly']
```

Comment:
We should generally not edit these files manually, they are managed by package butler and any change we make manually here will be undone by package butler down the road.

Reply:
Oops, sorry! Didn't realize they were automatically generated.

Reply:
No harm done, it will just disappear at some point again :)